spec/requests/case_logs_controller_spec.rb

```diff
@@ -275,21 +277,25 @@
                                 managing_organisation: organisation)
             end
             let!(:case_log_2022) do
-              FactoryBot.create(:case_log, :completed,
-                                owning_organisation: organisation,
-                                mrcdate: Time.zone.local(2022, 2, 1),
-                                startdate: Time.zone.local(2022, 12, 1),
-                                tenancy: 6,
-                                managing_organisation: organisation)
+              case_log = FactoryBot.build(:case_log, :completed,
+                                          owning_organisation: organisation,
+                                          mrcdate: Time.zone.local(2022, 2, 1),
+                                          startdate: Time.zone.local(2022, 12, 1),
+                                          tenancy: 6,
+                                          managing_organisation: organisation)
+              case_log.save!(validate: false)
```

Comment:
Any reason not to use the time cop pattern here?

Reply:
It's creating both a 21 and 22 case log and I wasn't sure I could freeze timecop to 2 seperate times

Reply:
Shouldn't any '21 log pass that validation anyway? Little worried this might cover other validation failures.

On a related note we may have a bit of a timebomb in our specs in general with the collection window validation? Once the '21 window closes presumably all logs we create in specs become invalid?

Reply:
I hadn't considered freezing it to the later date yesterday 🤦